go/test/endtoend/vreplication/vdiff_helper_test.go

```diff
@@ -71,7 +71,8 @@ func doVtctlclientVDiff(t *testing.T, keyspace, workflow, cells string, want *ex
 	ksWorkflow := fmt.Sprintf("%s.%s", keyspace, workflow)
 	t.Run(fmt.Sprintf("vtctlclient vdiff %s", ksWorkflow), func(t *testing.T) {
 		// update-table-stats is needed in order to test progress reports.
-		uuid, _ := performVDiff2Action(t, true, ksWorkflow, cells, "create", "", false, "--auto-retry", "--update-table-stats")
+		uuid, _ := performVDiff2Action(t, true, ksWorkflow, cells, "create", "", false, "--auto-retry",
+			"--update-table-stats", fmt.Sprintf("--filtered-replication-wait-time=%v", vdiffTimeout/2))
```

Comment:
You can see the reason for vdiff test changes in the commit message: https://github.com/vitessio/vitess/commit/46d733b7714be614c3f2caac90af25ca0993586d